src/main/java/org/abstractj/kalium/NaCl.java

```diff
@@ -87,6 +87,32 @@ public int crypto_generichash_blake2b_salt_personal(@Out byte[] buffer, @u_int64
                                                             @In byte[] salt,
                                                             @In byte[] personal);
 
```

Comment:
The master isn't ahead so there was nothing to rebase. I removed the unused imports.